larray/tests/test_session.py

```diff
@@ -275,44 +275,40 @@ def test_to_globals(self):
         assert_array_nan_equal(e, self.session.e)
         self.assertFalse(larray_equal(e, backup_value))
 
-    def test_eq(self):
+    def test_equals(self):
         sess = self.session.filter(kind=LArray)
         expected = Session([('e', self.e), ('f', self.f), ('g', self.g)])
-        self.assertTrue(all(sess == expected))
+        assert all(sess.equals(expected))
 
         other = Session({'e': self.e, 'f': self.f})
-        res = sess == other
-        self.assertEqual(res.ndim, 1)
-        self.assertEqual(res.axes.names, ['name'])
-        self.assertTrue(np.array_equal(res.axes.labels[0], ['e', 'g', 'f']))
-        self.assertEqual(list(res), [True, False, True])
+        res = sess.equals(other)
+        assert res.ndim == 1
+        assert res.axes.names == ['name']
+        assert np.array_equal(res.axes.labels[0], ['e', 'g', 'f'])
+        assert list(res) == [True, False, True]
 
         e2 = self.e.copy()
         e2.i[1, 1] = 42
         other = Session({'e': e2, 'f': self.f})
-        res = sess == other
-        self.assertEqual(res.axes.names, ['name'])
-        self.assertTrue(np.array_equal(res.axes.labels[0], ['e', 'g', 'f']))
-        self.assertEqual(list(res), [False, False, True])
+        res = sess.equals(other)
+        assert res.axes.names == ['name']
+        assert np.array_equal(res.axes.labels[0], ['e', 'g', 'f'])
+        assert list(res) == [False, False, True]
 
-    def test_ne(self):
+    def test_eq(self):
         sess = self.session.filter(kind=LArray)
         expected = Session([('e', self.e), ('f', self.f), ('g', self.g)])
-        self.assertFalse(any(sess != expected))
+        assert sess == expected
```

Comment:
the test is wrong and should be expanded to actually test that the result is correct (as explained in the changelog)